<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Resources/Sage_Corps_app_2.ico</normaloff>Resources/Sage_Corps_app_2.ico</iconset>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/background/FA4-1.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"> background-color:rgb(177,198,200);setWindowIcon:url(:/newPrefix/Sage_Corps_app.png);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 0, 0);
font: 25 20pt &quot;Microsoft JhengHei Light&quot;;</string>
   </property>
   <property name="text">
    <string>Medicine ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 0, 0);
font: 25 20pt &quot;Microsoft JhengHei Light&quot;;</string>
   </property>
   <property name="text">
    <string>Dosage (mg):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 0, 0);
font: 25 20pt &quot;Microsoft JhengHei Light&quot;;</string>
   </property>
   <property name="text">
    <string>Dispensed Amount: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 0, 0);
font: 25 20pt &quot;Microsoft JhengHei Light&quot;;</string>
   </property>
   <property name="text">
    <string>Frequency (per day): </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="medicine_id">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255); font-size:15pt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dosage">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255); font-size:15pt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dispensed_amount">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255); font-size:15pt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="frequency">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>220</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255); font-size:15pt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="qrButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(144, 191, 200); font-size:14pt;</string>
   </property>
   <property name="text">
    <string>Create QR Code</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="background.qrc"/>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
